pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -97,7 +97,12 @@ func InstallPlugin(pluginName, version string, c utils.CommandLine, client utils
 
 		// Plugins which are downloaded just as sourcecode zipball from github do not have checksum
 		if v.Arch != nil {
-			checksum = v.Arch[osAndArchString()].SHA256
+			archMeta, exists := v.Arch[osAndArchString()]
+			if !exists {
+				archMeta = v.Arch["any"]
+			}
+			checksum = archMeta.SHA256
+			}
```

Comment:
Looks like my previous suggestion included an extra closing curly brace, which fails to compile, sorry about that.
```suggestion
			checksum = archMeta.SHA256
```